### PR TITLE
Translation for the page 'Create a New React App'

### LINKED ISSUE #41: Translation for the page 'Installation -> Create a New React App'
# Please help us translating this page in Italian!
  
  **Aiutaci a tradurre questa pagina in Italiano!**

  You can find all the details about how to help here: https://github.com/reactjs/it.reactjs.org/issues/1
  
  ## GRAZIE! :smile:

### PR DESCRIPTION
**Pull request by LucaBlackDragon**
Closes #41

### CODE REVIEW

File: content/docs/create-a-new-react-app.md [hunk 1]
```diff
@@ -8,87 +8,87 @@ prev: add-react-to-a-website.html
...
 
-* A **compiler** such as [Babel](https://babeljs.io/). It lets you write modern JavaScript code that still works in older browsers.
+* Un **compilatore** come [Babel](https://babeljs.io/). Ti consente di scrivere codice JavaScript moderno che funziona anche nei vecchi browser.
 
-If you prefer to set up your own JavaScript toolchain from scratch, [check out this guide](https://blog.usejournal.com/creating-a-react-app-from-scratch-f3c693b84658) that re-creates some of the Create React App functionality.
+Se preferisci realizzare la tua toolchain JavaScript da zero, [leggi questa guida](https://blog.usejournal.com/creating-a-react-app-from-scratch-f3c693b84658) in cui alcune delle funzionalità di Create React App vengono ricreate.
 
-Don't forget to ensure your custom toolchain [is correctly set up for production](/docs/optimizing-performance.html#use-the-production-build).
+Non ti dimenticare di assicurarti che la tua toolchain personalizzata [sia impostata correttamente per la produzione](/docs/optimizing-performance.html#use-the-production-build).
```
Comment (deblasis) [inline content/docs/create-a-new-react-app.md]:
```suggestion
Non dimenticare di assicurarti che la tua toolchain personalizzata [sia impostata correttamente per la produzione](/docs/optimizing-performance.html#use-the-production-build).
```
I found just this tiny grammar typo! 👍

Comment (LucaBlackDragon) [reply content/docs/create-a-new-react-app.md]: Hi Alessandro, actually the "ti" is already included in "assicurar... **ti**".
If you think it's better, we could go for "Non ti dimenticare di controllare che la tua toolchain..."

Comment (deblasis) [reply content/docs/create-a-new-react-app.md]: Hi Luca! Sorry for the delay. Yeah, the point is that we should have one "ti" not two. I like your suggestion even more actually: 
"Non ti dimenticare di controllare che la tua toolchain...."

Comment (LucaBlackDragon) [reply content/docs/create-a-new-react-app.md]: I confused the correction and the original translation 🤦‍♂️
Anyway, fixed! THANKS!!!

**Pull request merged**